src/dashboard/containers/PersonalStepsAddedChart/__tests__/PersonalStepsAddedChart.js

```
@@ -10,38 +10,30 @@ describe('<PersonalStepsAddedChart />', () => {
         renderWithContext(<PersonalStepsAddedChart />).snapshot();
     });
 
-    /* It has to wait until this query is available in the schema
+    /*
     it('should render properly with chart', async () => {
         const { snapshot, getByText } = renderWithContext(
             <PersonalStepsAddedChart />,
             {
                 mocks: {
                     Query: () => ({
-                        stages_report: () => ({
-                            data: () => ([
-                                {
-                                    __typename: 'Data',
-                                    pathway_stage: 'NO STAGE',
-                                    member_count: 10,
-                                    steps_added_count: 12,
-                                    steps_completed_count: 37,
-                                },
-                                {
-                                    __typename: 'Data',
-                                    pathway_stage: 'NOT SURE',
-                                    member_count: 11,
-                                    steps_added_count: 34,
-                                    steps_completed_count: 37,
-                                }
-                            ])
-                        })
-                    })
+                        OrganizationStageReport: () =>  ([
```

I was trying for a while to mock response from Apollo Client but failed. I looked at your code in React Native app and it looks similar. Maybe you know what may be wrong here? I'd really appreciate that.

---

If you go through `Query`, you have to use the relationship names instead of the type names. So
```diff
- OrganizationStageReport: () =>  ([
+ organizationPathwayStagesReport: () =>  ([
```

I wish it would throw some more errors when you try to mock something that doesn't exist...

If you want a bunch of the same thing you can use https://www.apollographql.com/docs/graphql-tools/mocking/#using-mocklist-in-resolvers. 

You could also mock the type instead of the `Query` type if you wanted them to all be the same or be a random number:
```ts
mocks: {
  OrganizationStageReport: () => ({ memberCount: 1 }) // or some faker random value that would be different for each one...
}
```